Sources/NIOSSL/UniversalBootstrapSupport.swift

```diff
@@ -36,17 +36,30 @@ public struct NIOSSLClientTLSProvider<Bootstrap: NIOClientTCPBootstrapProtocol>:
     let context: NIOSSLContext
     let serverHostname: String?
     let customVerificationCallback: NIOSSLCustomVerificationCallback?
+    let additionalPeerCertificateVerificationCallback: _NIOAdditionalPeerCertificateVerificationCallback?
```

Comment:
Can we revert this change? I don't think we need it, so I'm inclined to discard it for now.

Reply:
Everything is internal only and we need it for tests or we need to rewrite them quite a bit I guess. Here is where we currently need it:
https://github.com/apple/swift-nio-ssl/blob/22bc6f6e4b5ee7487e5fdf0f23cecd16f5bae7ea/Tests/NIOSSLTests/NIOSSLIntegrationTest.swift#L293-L307

Reply:
I hadn't realised that we were using the TLS provider API for all our testing. Ok, we can keep this change.